main.c

```diff
@@ -623,6 +663,12 @@ int main(int argc, char *argv[]) {
 		return EXIT_FAILURE;
 	}
 
+	if ((state.xkb_context = xkb_context_new(XKB_CONTEXT_NO_FLAGS)) == NULL) {
+		// TODO for reviewer: Silent failure? or smth else?
```

Comment:
"xkb_context_new failed" printed to stderr is fine

Reply:
ok, I pushed a commit for that. Also, it returns 1 (i.e. exits)